oaknational/Oak-Web-Application
# Refactor/subject listing use new api - LESQ-730

## Description

Music year: {owa_music_year}

- remove query to legacy api on subject listing page
- update tests to use the 2023 fixture


1. Go to https://deploy-preview-2347--oak-web-application.netlify.thenational.academy

How to test
Go to:
Click on subject listing page for different keystages
You should see everything working as expected

## Checklist
